input/fsh/modules/files.fsh

```diff
@@ -128,6 +142,14 @@ Description: "Information about a file related to a research participant"
 * extension[file-format] ^short = "The file format used (EDAM is preferred)"
 * content.attachment.url 1..1 /*Location uri*/
 * content.attachment.url ^short = "The URI at which this data can be accessed"
+* content ^slicing.discriminator.type = #pattern
+* content ^slicing.discriminator.path = "code"
+* content ^slicing.rules = #openAtEnd
+* content ^slicing.ordered = false

```

Comment:
I'm not sure we need slicing at this point since we are fine with regular attachments.

Reply:
Right now we have:
```
* content.attachment.url 1..1 /*Location uri*/
* content.attachment.url ^short = "The URI at which this data can be accessed"
```
and this made me think there was only one attachment url you could make, thus why I tried doing the slicing. But content is 1..* which inherently means one could attach more than one. Do you want to change how we write the profile or do you think it's self-explanatory?

Reply:
So, the tricky part is that content is an array of 1 or more things, each of which will contain one attachment. Each of those attachments must have 1 and only 1 URL. It is a bit confusing, but it does allow multiple URLs, but only because you can have more than one thing inside the content array.